CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+## v2.10.0 / 2023-08-31
+
+### Note
+- By default experimental label and annotation metrics are disabled now to reduce memory usage. Before this change, they would include name and namespace labels.
+
+* [BUGFIX] custommetrics: log no _info suffix in name only once per reading the configuration by @chrischdi in https://github.com/kubernetes/kube-state-metrics/pull/2157
+* [BUGFIX] Don't crash on non-existent valueFrom path values by @chihshenghuang in https://github.com/kubernetes/kube-state-metrics/pull/2140
+* [BUGFIX] Index out of range in metrics_store.SanitizeHeaders by @mrueg in https://github.com/kubernetes/kube-state-metrics/pull/2166
+* [BUGFIX] custommetrics: always extract the headers but only write it when we have metrics by @chrischdi in https://github.com/kubernetes/kube-state-metrics/pull/2154
```

Comment:
These are all related to custom metrics so it feels inconsistent to have some prefixed by `custommetrics:` and some not

Reply:
I was using the PR title as generated by github when drafting a release.

Reply:
I updated the titles a bit.

Reply:
ok